.github/workflows/labels.yml

```diff
@@ -0,0 +1,12 @@
+name: Labeler
+on: [pull_request]
+
+jobs:
+  label:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/labeler@v2
```

Comment:
Where are those labels configured 🤔?

<img width="238" alt="Screenshot 2020-01-15 at 10 12 04" src="https://user-images.githubusercontent.com/2358722/72420747-95b0fd00-377f-11ea-9eb2-e4d4503e88b6.png">

Reply:
I'd also say that `api-non-breaking-change` is too much. As most of PRs will be labelled with this one it's introducing some noise. Maybe we can add only `api-breaking-change` when needed?

Reply:
The labels can be configured manually in the Github project directly.